geode-core/src/main/java/org/apache/geode/management/internal/SystemManagementService.java

```diff
@@ -690,6 +705,14 @@ private SystemManagementService init() {
     }
   }
 
+  private static LocalManager createLocalManager(SystemManagementService service) {
```

Comment:
are we doing this for testing? some of the changes seem harder to understand than the old code.

Reply:
Yes, this is to make it testable. The changes in `startManager()` are the only changes required to fix the LGTM warning. Every other change is to make this class testable.